### ISSUE TITLE
apidocs not working

### ISSUE THREAD

**Issue by stefanwalther** [verbose/verb-cli#1]
Hi,

I have done the following but I don't get apidocs to work:

- Changed the path in .verb.md ```{%= apidocs("./lib/qrs.js") %}```

- Added the following line to ~\AppData\Roaming\nvm\v0.12.0\node_modules\verb-cli\node_modules\verb-default\index.js

```javascript
verb.helper('apidocs', require('helper-apidocs'));
```

So the final result is:

'use strict';

var gutil = require('gulp-util');

```javascript
module.exports = function (verb) {
	verb.helper('apidocs', require('helper-apidocs'));
  verb.task('default', function() {
    verb.src(['.verb*.md', 'docs/_verb/**/*.md'])
      .on('error', gutil.log)
      .pipe(verb.dest('./'));
  });
  return verb;
};
```

Any ideas? What do I miss?

**Comment by jonschlinkert** [comment]
the `apidocs` helper is built-in to verb, was it not working without registering a custom helper? I had some issues with it this past week too, so if you need to use it standalone I can prioritize a fix for it

**Comment by jonschlinkert** [comment]
btw, you don't need the `module.exports` code unless you're exporting that as a module. 

you can just do:

```
'use strict';
var gutil = require('gulp-util');
var verb = require('verb');

verb.helper('apidocs', require('helper-apidocs'));
verb.task('default', function () {
  verb.src(['.verb.md', 'docs/_verb/**/*.md'])
    .on('error', gutil.log)
    .pipe(verb.dest('./'));
});
```

**Comment by stefanwalther** [comment]
Thx, ;-) the module.exports was in your code https://github.com/verbose/verb-default/blob/master/index.js, just copied it and inserted the line ...

No, it didn't work by just using verb resp. verb-cli ...

**Comment by jonschlinkert** [comment]
yeah, the module.exports is in that code since it's being used as a module by verb-cli. But when you have a `verbfile.js` in your project, you can just use verb like you would gulp. 


> No, it didn't work by just using verb resp. verb-cli ...

hmm, did you try clearing your npm cache and reinstalling verb-cli?

**Comment by stefanwalther** [comment]
I'll give it a try ...

**Comment by jonschlinkert** [comment]
definitely let me know if that doesn't solve it and I'll look into whatever is causing issues

**Comment by stefanwalther** [comment]
Tried to clear npm cache, re-installed verb-cli globally and updated verb in the local project, now I get:

```bash
D:\Projects\Node.js\verb-cli\lib\handler.js:96
  verb.init(context);
       ^
TypeError: undefined is not a function
    at Liftoff.module.exports (D:\Projects\Node.js\verb-cli\lib\handler.js:96:8)
    at Liftoff.launch (D:\Projects\Node.js\verb-cli\node_modules\liftoff\index.js:144:6)
    at Object.<anonymous> (D:\Projects\Node.js\verb-cli\bin\verb:22:5)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3

```

Will try to find out more ... and post it here ...

**Comment by stefanwalther** [comment]
Hi, another observation:
I realized that this is not related to
```verb.helper('apidocs', require('helper-apidocs'));```

I have uninstalled all verb components again but verb by default throws this error (on a clean system). Trying to debug but didn't succeed yet ...

Regards
Stefan

**Comment by jonschlinkert** [comment]
I'm still working on this. it's a bug, I'll have it fixed today I think

**Comment by stefanwalther** [comment]
OK, sorry, didn't want to be too picky, just trying to understand verb and all the dependencies ... ;-)

I'll write down my findings, may be a good starting point for some kind of documentation ...

**Comment by jonschlinkert** [comment]
> OK, sorry, didn't want to be too picky

never took it that way! please feel comfortable creating issues for anything you find. any feedback is great
